superset-frontend/src/components/FilterableTable/index.tsx

```diff
@@ -385,31 +366,6 @@ const FilterableTable = ({
     return className;
   };
 
-  const sort = ({
-    sortBy,
-    sortDirection,
-  }: {
-    sortBy: string;
-    sortDirection: SortDirectionType;
-  }) => {
-    const shouldClearSort =
-      sortDirectionState === SortDirection.DESC && sortByState === sortBy;
-
-    if (shouldClearSort) {
-      setSortByState(undefined);
-      setSortDirectionState(undefined);
-      setDisplayedList([...list]);
-    } else {
-      setSortByState(sortBy);
-      setSortDirectionState(sortDirection);
-      setDisplayedList(
-        [...list].sort(
-          sortResults(sortBy, sortDirection === SortDirection.DESC),
-        ),
-      );
-    }
-  };
-
   const addJsonModal = (
```

Comment:
Can you rename `addJsonModal` to `jsonModal` or `newJsonModal`. This function is not adding anything, it's returning a modal instance.